sensor/kubernetes/fake/flows.go

```diff
@@ -210,18 +235,72 @@ func makeNetworkConnection(src string, dst string, containerID string, closeTime
 	}
 }
 
-func (w *WorkloadManager) getFakeNetworkConnectionInfo(workload NetworkWorkload) *sensor.NetworkConnectionInfo {
-	conns := make([]*sensor.NetworkConnection, 0, workload.BatchSize)
-	networkEndpoints := make([]*sensor.NetworkEndpoint, 0, workload.BatchSize)
-	for i := 0; i < workload.BatchSize; i++ {
-		src, ok := ipPool.randomElem()
+// Randomly decide to get an interal or external IP. If the IP is external
+// randomly decide to pick it from a pool of external IPs or a new exteranl IP.
+// We want to have cases where multiple different entities connect to the same
+// external IP, but we also want a large number of unique external IPs.
```

Comment:
Might be worth adding to this comment to explain the probabilities of each case - as I understand it:
- 80% chance to be internal
- 20% chance to be external
   - 50/50 chance of that external IP being from the pool or new

Reply:
Done